lib/vanagon/component/source/http.rb

```diff
@@ -105,7 +105,7 @@ def extract(tar)
             end
           else
             # Extension does not appear to be an archive
-            return nil
+            return ':'
```

Comment:
soooo, it looks like we're divided... I'm just going to leave this as is until someone tells me definitely to change it.